apps/box.go

```diff
@@ -0,0 +1,39 @@
+package apps
+
+import (
+	"encoding/binary"
+	"fmt"
+)
+
+const boxPrefix = "bx:"
+const boxPrefixLength = len(boxPrefix)
+const boxNameIndex = boxPrefixLength + 8 // len("bx:") + 8 (appIdx, big-endian)
+
+// MakeBoxKey creates the key that a box named `name` under app `appIdx` should use.
+func MakeBoxKey(appIdx uint64, name string) string {
```

Comment:
@AlgoStephenAkiki Was it a by-design decision _not_ to move `TestMakeBoxKey` from go-algorand to here?

I ask because I'd expect the test to move with the functionality.

Reply:
I have the same question RE:  `TestNewAppCallBytes`.

Reply:
No that was an error on my part.  Good catch...missed it.

Reply:
@AlgoStephenAkiki Resolving - Thanks for migrating the tests.